test/functional/driver/motion-e2e-specs.js

```diff
@@ -47,29 +47,26 @@ describe('ReduceMotion', function () {
     await shutdownSimulator(sim);
   });
 
+  async function getReduceMotion (driver) {
```

Comment:
@vrunoa FYI

Reply:
Thanks for the improvement @mykola-mokhnach